### PR TITLE
for comment

### PR DESCRIPTION
**Pull request by zhuoshen617**

### CODE REVIEW

File: scheduler.py [hunk 1]
```diff
@@ -1,128 +0,0 @@
...
-    collection = client["findMyGME"]["stocks"]
-
-    # Creates an instance of the reddit object
-    reddit = praw.Reddit(client_id=os.environ['client_id'], client_secret=os.environ['client_secret'],
-                         user_agent=os.environ['user_agent'])
-
-    # A list of abbreviations that generally do not represent tickers.
-    black_list = ["A", "DD", "FOR", "CEO", "ALL", "EV", "OR", "AT", "RH", "ONE", "ARE", "VERY", "ON", "EDIT"]
-
-    main()
```
Comment (zhuoshen617) [inline scheduler.py:128]: move everything other than main() into main()

File: scheduler.py [hunk 2]
```diff
@@ -1,128 +0,0 @@
...
-    load_dotenv()
-    client = MongoClient(os.environ['DB_URL'])
-    collection = client["findMyGME"]["stocks"]
-
-    # Creates an instance of the reddit object
-    reddit = praw.Reddit(client_id=os.environ['client_id'], client_secret=os.environ['client_secret'],
-                         user_agent=os.environ['user_agent'])
-
-    # A list of abbreviations that generally do not represent tickers.
-    black_list = ["A", "DD", "FOR", "CEO", "ALL", "EV", "OR", "AT", "RH", "ONE", "ARE", "VERY", "ON", "EDIT"]
```
Comment (zhuoshen617) [inline scheduler.py:126]: Create a constant on top of the file
TICKER_BLACKLIST = ["A", "DD", "FOR", "CEO", "ALL", "EV", "OR", "AT", "RH", "ONE", "ARE", "VERY", "ON", "EDIT"]

File: scheduler.py [hunk 3]
```diff
@@ -1,128 +0,0 @@
-import time
```
Comment (zhuoshen617) [inline scheduler.py:1]: order your import alphabetically. remove all blank lines in between.

Put this file in pycharm and the ones greyed out can be removed.

File: scheduler.py [hunk 4]
```diff
@@ -1,128 +0,0 @@
...
-
-import yfinance as yf
-from dotenv import load_dotenv
-
-import dotenv
-import os
-
-
-# Reads, parses and converts the ticker files into dictionary
-def build_dictionary(file_name, ticker_dict):
```
Comment (zhuoshen617) [inline scheduler.py:17]: `def build_dictionary(file_name, ticker_dict):
    with open(file_name, 'r') as file:
        for line in file:
            line_list = line.strip().split("\t")
            if len(line_list) < 2:
                continue

            symbol = line_list[0]
            name = line_list[1]
            if symbol not in ticker_dict:
                    ticker_dict[symbol] = name`

File: scheduler.py [hunk 5]
```diff
@@ -1,128 +0,0 @@
...
-    with open(file_name, 'r') as file:
-        for line in file:
-            line_list = line.strip().split("\t")
-            if len(line_list) >= 2:
```
Comment (zhuoshen617) [inline scheduler.py:21]:
```
if len(line_list) < 2:
    continue
```

File: scheduler.py [hunk 6]
```diff
@@ -1,128 +0,0 @@
...
-
-    print(toDB)
-
-    collection.insert_many(toDB)
-
-
-def main():
-    # Creates a combined NASDAQ and NYSE stock ticker dictionary
-    ticker_dict = {}
-    build_dictionary("Ticker Symbols/NASDAQ.txt", ticker_dict)
```
Comment (zhuoshen617) [inline scheduler.py:98]: rename your directory to tickers/NASDAQ ...

File: scheduler.py [hunk 7]
```diff
@@ -1,128 +0,0 @@
...
-    build_dictionary("Ticker Symbols/NYSE.txt", ticker_dict)
-
-    # Counts the appearances of tickers in the top500 hot posts in WSB
-    sorted_ticker_count = count_tickers(reddit, ticker_dict)
-
-    # Filters out the abbreviations that generally do not represent stock tickers
-    # Also cuts down the size of the dictionary to 10
-    filtered_sorted_ticker_dict = filter_dictionary(sorted_ticker_count, black_list)
```
Comment (zhuoshen617) [inline scheduler.py:106]: why do a post filtering. Don't add to dict if it is on blacklist.

File: scheduler.py [hunk 8]
```diff
@@ -1,128 +0,0 @@
...
-            "industry": stock_info["industry"],
-            "count": output_count,
-            "previousClose": stock_info["previousClose"],
-            "fiftyDayAverage": stock_info["fiftyDayAverage"],
-            "averageDailyVolume10Day": stock_info["averageDailyVolume10Day"],
-            "timeStamp": now
-        }
-        toDB.append(entry)
-
-    print(toDB)
```
Comment (zhuoshen617) [inline scheduler.py:90]: remove prints

File: scheduler.py [hunk 9]
```diff
@@ -1,128 +0,0 @@
...
-
-
-# Packages the top10 tickers and related financial information into dictionaries
-# and inserts them into the database with time stamps
-def insert_to_db(sorted_ticker_list, filtered_sorted_ticker_dict, ticker_dict):
-    toDB = []
-
-    now = datetime.now()
-
-    for i in range(10):
```
Comment (zhuoshen617) [inline scheduler.py:73]: put 10 in a constant

File: scheduler.py (hunk 3 above)
Comment (LilyXueLi) [reply scheduler.py:1]: to be confirmed

File: scheduler.py (hunk 9 above)
Comment (LilyXueLi) [reply scheduler.py:73]: to discuss

File: scheduler.py (hunk 6 above)
Comment (LilyXueLi) [reply scheduler.py:98]: to discuss

**Pull request closed without merging**